### PR TITLE
Add theme sharing tutorial to guide

### PR DESCRIPTION
**Pull request by freddyaboulton**
# Description

Add theme sharing tutorial to guide

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have added a short summary of my change to the CHANGELOG.md
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# A note about the CHANGELOG

Hello 👋 and thank you for contributing to Gradio!

All pull requests must update the change log located in CHANGELOG.md, unless the pull request is labeled with the "no-changelog-update" label.

Please add a brief summary of the change to the Upcoming Release > Full Changelog section of the CHANGELOG.md file and include
a link to the PR (formatted in markdown) and a link to your github profile (if you like). For example, "* Added a cool new feature by `[@myusername](link-to-your-github-profile)` in `[PR 11111](https://github.com/gradio-app/gradio/pull/11111)`".

If you would like to elaborate on your change further, feel free to include a longer explanation in the other sections.
If you would like an image/gif/video showcasing your feature, it may be best to edit the CHANGELOG file using the 
GitHub web UI since that lets you upload files directly via drag-and-drop.

### CODE REVIEW

File: guides/06_other-tutorials/theming-guide.md [hunk 1]
```diff
@@ -256,4 +256,95 @@ Look how fun our theme looks now! With just a few variable changes, our theme lo
...
+
+<iframe
+	src="https://gradio-theme-gallery.hf.space"
+	frameborder="0"
+	width="850"
+	height="450"
+></iframe>
+
+### Downloading
+To use a theme from the hub, use the `from_hub` method on the `ThemeClass` and pass it to your app:
```
Comment (abidlabs) [inline guides/06_other-tutorials/theming-guide.md:329]:
```suggestion
To use a theme from the hub, use the `from_hub` method on the `ThemeClass` and pass the name of the theme Space in the format (username/space_name) to your app:
```

**Comment by abidlabs** [comment]
Love it! Theme sharing is so cool

**Review by abidlabs — APPROVED** [review]

**Comment by abidlabs** [comment]
Unrelated to your PR @freddyaboulton but I was reading through this Guide again, and I think it would be good to list the 6 built-in themes after this paragraph:

> Gradio comes with a set of prebuilt themes which you can load from `gr.themes.*`. Each of these themes set values for hundreds of CSS variables.

Since you're working on this Guide, do you wanna make this change?

**Comment by freddyaboulton** [comment]
Yea abosolutely @abidlabs !

File: guides/06_other-tutorials/theming-guide.md [hunk 2]
```diff
@@ -17,10 +17,15 @@ with gr.Blocks(theme=gr.themes.Soft()):
 	height="450"
 ></iframe>
 
+Gradio comes with a set of prebuilt themes which you can load from `gr.themes.*`. These are:
 
-Gradio comes with a set of prebuilt themes which you can load from `gr.themes.*`. Each of these themes set values for hundreds of CSS variables.
+* `gr.themes.Base()`
```
Comment (freddyaboulton) [inline guides/06_other-tutorials/theming-guide.md:22]: Added a blurb on all the themes that ship with gradio

image

Comment (abidlabs) [reply guides/06_other-tutorials/theming-guide.md:22]: Nice

**Comment by freddyaboulton** [comment]
Thank you for the review @abidlabs !

**Pull request merged**